specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json

```diff
@@ -0,0 +1,215 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "MicrosoftSerialConsoleClient",
+        "version": "2018-07-01-preview",
+        "description": "Azure Virtual Machine Serial Console allows you to access serial console of a Virtual Machine"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.SerialConsole/consoleServices/{default}": {
+            "get": {
+                "tags": [
+                    "ConsoleDisabled"
+                ],
+                "operationId": "ListConsole_Disabled",
+                "description": "Gets if Serial Console is disabled for a subscription.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "default",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "Default string modeled as parameter for URL to work correctly.",
+                        "enum": [
+                          "default"
+                        ]
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK - Returns a JSON object"
```

Comment:
Provided example has the `response 200` and it has `"body"` but the swagger has no `schema` for the `body` value.